### ISSUE TITLE
[Enhancement] Add opt-out for item reformatting for text resizing

### ISSUE THREAD

**Issue by RasmusKjeldgaard** [kirbydesign/designsystem#4520]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
When text is resized on mobile, we re-arrange the item layout, as seen here:

https://65ef73cd884a4ee74ff05d78-bbyzaugtgd.chromatic.com/?path=/story/components-item--item-text-resize-layout&globals=viewport.value:mobile1

Up until now, a special case for this was when items contain a control (checkbox/radio/toggle/button), in which case we do not apply new layout styles.

This has proven to be an issue for complex items that have a checkbox, where the label can become so squished by other elements that it becomes unusable. 

We should add a way for consumers to opt-in to the layout styles for the controls-based items as well.

## Describe the solution you'd like
I'd like a class that can be applied to items, that opts out of the text resize behavior. 

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-refinement)
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".
